web/src/components/customer/my-commits/CommitStatusBadge.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import {CommitStatus} from 'interfaces';
+import styled, {css} from 'styled-components';
+
+const dangerStyle = css`
```

Comment:
Maybe "actionStyle" is a better name?

Reply:
@xlai20 Okay will rename it to `pendingActionStyle` :D